src/cargo/core/resolver/resolve.rs

```diff
@@ -47,8 +47,8 @@ pub struct Resolve {
 }
 
 /// A version to indicate how a `Cargo.lock` should be serialized. Currently
-/// V2 is the default when creating a new lockfile. If a V1 lockfile already
-/// exists, it will stay as V1.
+/// [V3 is the default](ResolveVersion::default) when creating a new lockfile.
```

Comment:
Should we just not call out the latest version?  Seems like this will be easy to get out of date again